presto-yarn-package/src/main/slider/package/templates/queues.json.j2

```diff
@@ -0,0 +1,15 @@
+{
+  "queues": {
+    "global": {
+      "maxConcurrent": 10,

```

Comment:
It will be nice to have these properties in queue.json configurable in appConfig like others. Something like "site.global.queue_max_concurrent":"10" and use that here. But if you prefer you can just leave it as-is, and we will address it a separate patch if needed.